v4/export/dump.go

```diff
@@ -110,6 +112,44 @@ func dumpTable(ctx context.Context, conf *Config, db *sql.DB, dbName string, tab
 		return err
 	}
 
+	if conf.Rows != UnspecifiedSize {
```

Comment:
please put these codes into a new function to make main logic clear

Reply:
fixed